README.md

```diff
@@ -867,7 +869,15 @@ There are a few important caveats about the provided kdump support:
 
 ### NVIDIA GPUs Support
 Bottlerocket's `nvidia` variants include the required packages and configurations to leverage NVIDIA GPUs.
-The official AMIs for these variants can be used with EC2 GPU-equipped instance types such as: `p2`, `p3`, `p4`, `g4dn`, `g5` and `g5g`.
+Currently, the following NVIDIA driver versions are supported in Bottlerocket:
+
+* 470.X
+* 515.X
```

Comment:
Would it be helpful to add the corresponding Bottlerocket versions alongside the NVIDIA driver numbers? 

e.g.
```markdown
470.X: Bottlerocket v0.X - v0.Y
515.X: Bottlerocket v0.A - v0.B
```